docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -0,0 +1,338 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "opened-florist",
+   "metadata": {},
+   "source": [
+    "# Pegasos Quantum Support Vector Classifier\n",
+    "\n",
+    "There's another SVM based algorithm that benefits from the quantum kernel method. Here, we introduce an implementation of a another classification algorithm, which is an alternative version to the QSVC shown above. This classification algorithm implements the Pegasos algorithm from the paper \"Pegasos: Primal Estimated sub-GrAdient SOlver for SVM\" by Shalev-Shwartz et al., see: https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf.\n",
+    "\n",
+    "This algorithm:\n",
```

Comment:
I would remove this paragraph and integrate it into the next one (see suggestions below)

Reply:
Slightly re-arrange the text.